packages/components/src/Notification/subcomponents/GenericNotification/GenericNotification.tsx

```diff
@@ -82,7 +112,8 @@ export const GenericNotification = ({
       {...restProps}
     >
       <div className={styles.icon}>
-        <NotificationIcon type={type} />
+        {type && <NotificationIconType type={type} />}
+        {variant && <NotificationIconVariant variant={variant} />}
```

Comment:
Probably safer this way
```suggestion
        {type ? <NotificationIconType type={type} /> : <NotificationIconVariant variant={variant} />}
```